execution_engine/src/lib.rs

```diff
@@ -18,3 +18,52 @@ pub mod core;
 pub mod shared;
 /// Storage for the execution engine.
 pub mod storage;
+
+/// Relative location (to storage) where rocksdb data will be stored.
+pub const ROCKS_DB_DATA_DIR: &str = "rocksdb-data";
+
+const ROCKS_DB_BLOCK_SIZE_BYTES: usize = 256 * 1024;
+const ROCKS_DB_COMPRESSION_TYPE: rocksdb::DBCompressionType = rocksdb::DBCompressionType::Zstd;
+const ROCKS_DB_COMPACTION_STYLE: rocksdb::DBCompactionStyle = rocksdb::DBCompactionStyle::Level;
+const ROCKS_DB_ZSTD_MAX_DICT_BYTES: i32 = 256 * 1024;
+const ROCKS_DB_MAX_LEVEL_FILE_SIZE_BYTES: u64 = 512 * 1024 * 1024;
+const ROCKS_DB_MAX_OPEN_FILES: i32 = 768;
+const ROCKS_DB_ZSTD_COMPRESSION_LEVEL: i32 = 3; // Default compression level
+const ROCKS_DB_ZSTD_STRATEGY: i32 = 4; // 4: Lazy
+const ROCKS_DB_WAL_BITS: i32 = -14;
+
+/// Default constructor for rocksdb options.
+pub fn rocksdb_defaults() -> rocksdb::Options {
```

Comment:
I don't want to do this because consumers of this library may still want to open the database that is created with these options. That includes tools that shouldn't have to know how to deal with a hypothetical DbGlobalStateOptions type.